utils/desktop_bridge.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+let callback;
+
+window.desktopBridge = {};
+
+export function connect(callbackFunction) {
+    // store provided callback for later use
+    callback = callbackFunction;
+
+    // connect bridge methods
+    window.desktopBridge.updateUserActivityStatus = updateUserActivityStatus;
+}
+
+const updateUserActivityStatus = (userIsActive) => {
+    callback('updateUserActivityStatus', {userIsActive});
```

Comment:
I know we check in the desktop app if `updateUserActivityStatus` is `null` before calling it, implying that `callback` will have been set, but I suppose it's possible for `connect` to have been called with a `null` `callbackFunction`. 0/5 on doing anything about this.

Reply:
Redid the bridge ... this has changed.